docs/the-basics-console-mode.md

```diff
@@ -5,46 +5,46 @@ title: Console Mode
 
 ## Introduction
 
-Console Mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots on existing messaging platform (e.g. Messenger, LINE), it is recommended to build, test, and debug your bot in the console mode.
+The console mode is a productivity feature shipped with Bottender. It speeds up bot development iteration by running bots in the console. Comparing with testing your bots fully on the existing messaging platforms (e.g., Messenger, LINE), we recommend building, testing, and debugging part of your bots in the console mode.
 
-## Running in Console Mode
+## Running in the Console Mode
 
-In Bottender apps that created by Create Bottender App, you can pass `--console` option from npm scripts to underlying start/dev commands:
+In Bottender applications that created by Create Bottender App, you can provide the `--console` option from npm scripts to the underlying start/dev commands:
 
 ```sh
 npm start -- --console
 npm run dev -- --console
 ```
 
-With this option, the bot will be started and run in the console:
+With the `--console` option, Bottender runs the bot in the console:
```

Comment:
```suggestion
With `--console` option, Bottender runs the bot in console mode:
```

Reply:
`With the `--console` option, Bottender runs the bot in Console Mode:`